demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -49,16 +53,20 @@ class LintManager:
         verbose(int): Whether to output a detailed response.
         quiet(bool): Whether to output a quiet response.
         log_path(str): Path to all levels of logs.
+        prev_ver(str): Previous branch or SHA1 commit to run checks against.
+        json_file_path(str): Path to a json file to write the run resutls to.
+        id_set_path(str): Path to an existing id_set.json.
+        check_dependent_packs(bool): Whether to run lint also on the packs dependent on the given packs.
     """
 
     def __init__(self, input: str, git: bool, all_packs: bool, quiet: bool, verbose: int, prev_ver: str,
-                 json_file_path: str = ''):
+                 json_file_path: str = '', id_set_path: str = None, check_dependent_packs: bool = False):
 
         # Verbosity level
         self._verbose = not quiet if quiet else verbose
         # Gather facts for manager
         self._facts: dict = self._gather_facts()
-        self._prev_ver = prev_ver
+        self._prev_ver = prev_ver  # None if not inserted (and not master as was the default value)
```

Comment:
why did you change the default?

Reply:
The default value for `prev_ver` was set to `master`, which did not make any sense as the default case. In cases we run lint on master branch itself, the `prev_ver` we need to compare to is one previous commit (*which is case 1, and see note below). So, when the default value was set to `master`, the way to check if we are in case 1 was to check if `active_branch == master and prev ver == master`, which didn't make much sense (as it is not dependent on it being master - it depends on if it is specified or not). I modified the default value for the code to be more readable. 

Note: I could not find any way we use lint that hits case 1. When ever we run lint on master, we use lint -a. Should I still leave this case as is? (This case's functionality was present before my PR).

Reply:
So, after all that, i saw that in other similar cases in sdk repo the prev_ver is set to master as default, so I changed it back to the way it was originally.